sentences/fi/_common.yaml

```diff
@@ -40,14 +22,41 @@ lists:
         out: "blue"
       - in: "vihreä"
         out: "green"
+      - in: "violetti"
+        out: "purple"
+      - in: "ruskea"
+        out: "brown"
+  brightness:
+    range:
+      type: "percentage"
+      from: 0
+      to: 100
+  temperature:
+    range:
+      type: "temperature"
+      from: 0
+      to: 100
+  temperature_unit:
+    values:
+      - in: "(c|celsius)"
+        out: "celsius"
+      - in: "(f|fahrenheit)"
+        out: "fahrenheit"
```

Comment:
I kinda like the shortened version of celsius and fahrenheit here. In general on my other projects I do not recommend moving the code and changing it at the same time as it makes noticing changes harder. However I'm a newbie here, so that's just a thought of a random person, not something to panic about. 😄 

Why was the code order changed anyways? Did it match better the English or other versions? (that could be a good reason)